Resources/LaboratoryExchange-DiagnosticReport.xml

```diff
@@ -0,0 +1,250 @@
+      <comment value="In addition to the required category valueset, this element allows various categorization schemes based on the owner’s definition of the category and effectively multiple categories can be used at once. The level of granularity is defined by the category concepts in the value set." />
+      <alias value="ResultaatType" />
+      <min value="1" />
+      <mustSupport value="true" />
+      <binding>
+        <strength value="preferred" />
+        <description value="Codes for high level observation categories." />
+        <valueSet value="http://hl7.org/fhir/ValueSet/observation-category" />
+      </binding>
+    </element>
+    <element id="DiagnosticReport.category:laboratoryCategory">
+      <path value="DiagnosticReport.category" />
+      <sliceName value="laboratoryCategory" />
+      <short value="Classification of type of observation" />
+      <definition value="A code that classifies the general type of observation being made." />
+      <min value="1" />
+      <max value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://terminology.hl7.org/CodeSystem/v2-0074" />
+          <code value="LAB" />
+        </coding>
+      </patternCodeableConcept>
+      <binding>
+        <strength value="required" />
+        <description value="Code for Diagnostic Service Codes" />
+      </binding>
+    </element>
+    <element id="DiagnosticReport.category:resultType">
+      <path value="DiagnosticReport.category" />
+      <sliceName value="resultType" />
+      <short value="ResultType" />
+      <definition value="The type of result defines the laboratory specialty under which the test is categorized." />
+      <alias value="ResultaatType" />
+      <binding>
+        <strength value="required" />
+        <description value="Codes for high level observation categories." />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.13.1.1--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="lu-dataset-2" />
+        <map value="lu-concept-v2-4289" />
+        <comment value="ResultType" />
+      </mapping>
+    </element>
+    <element id="DiagnosticReport.code">
+      <path value="DiagnosticReport.code" />
+      <short value="PanelOrBattery" />
+      <definition value="For laboratory tests comprising multiple subtests and often requested together as a whole, this concept contains the name of the compound request (often indicated as a ‘panel’, ‘battery’ or ‘cluster’). Examples include: blood gases and EBV serology." />
+      <alias value="Onderzoek" />
+      <mustSupport value="true" />
+      <binding>
+        <strength value="extensible" />
+        <description value="PanelOrBatteryCodelist" />
+        <valueSet value="https://decor.nictiz.nl/ad/#/lu-/terminology/valueset/2.16.840.1.113883.2.4.3.11.60.40.2.13.1.5" />

```

Comment:
This is not the canonical of the ValueSet, it should be <http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.13.1.5--20200901000000>